src/utilities/getIntrospectionQuery.ts

```diff
@@ -32,6 +32,10 @@ export interface IntrospectionOptions {
    * Default: false
    */
   inputValueDeprecation?: boolean;
+
+  // Whether target GraphQL server supports `@oneOf` input objects.
+  // Default: false
+  inputObjectOneOf?: boolean;
```

Comment:
Followed the pattern here above for making this configurable, but wasn't sure if that was necessary.